asllib/doc/asl.spec

```diff
@@ -328,37 +328,37 @@ operator implies(Bool, Bool) -> Bool
   math_macro = \implies,
 };
 
-operator int_plus(list1(N)) -> N
+operator num_plus(list1(Z)) -> Z
```

Comment:
nit: this seems a bit odd, in that the name has gone from `int` --> `num` but the type has gone from `num` --> `int`?